### PR TITLE
One more step to dropping jsPlumb entirely

### LINKED ISSUE #64: create replacement for jsPlumb.draggable
Here is how we use it to make blocks draggable:
```
jsPlumb.draggable(dom, {
            grid: [size, size],
            containment: "parent",
            drag: function(event) {
                conn.redrawAll();
            },
            stop: function (event) {
                let x = event.pos[0]-$(".Sidebar").innerWidth();
                let y = event.pos[1];
                Session.move(`${App.selected()}/Blocks/${vnode.dom.dataset.idx}`, x, y);
            }
        });
```
We need a function that works basically the same so we can drop jsPlumb entirely.

### PR DESCRIPTION
**Pull request by maxknivets**
This should close #64.

### CODE REVIEW

**Comment by progrium** [comment]
Hah! Should have known, jQuery to the rescue.

**Comment by progrium** [comment]
Seems to cause a crash now. Apparently x,y aren't being sent properly:
```
2020/07/17 11:09:10 set: /Package/Declarations/0/Function/Blocks/0/Position [<nil> <nil>]
```
looking into it

File: ui/lib/app.js [hunk 1]
```diff
@@ -88,15 +88,15 @@ class App {
...
+        $("#" + dom.id).draggable({
             grid: [size, size],
             containment: "parent",
             drag: function(event) {
                 conn.redrawAll();
             },
             stop: function (event) {
-                let x = event.pos[0]-$(".Sidebar").innerWidth();
-                let y = event.pos[1];
+                let x = parseInt(event.target.style.left.replace("px","")) - $(".Sidebar").innerWidth() + "px";
```
Comment (progrium) [inline ui/lib/app.js:98]: we don't actually want "px"

File: ui/lib/app.js [hunk 2]
```diff
@@ -88,15 +88,15 @@ class App {
...
+                let y = event.target.style.top;
```
Comment (progrium) [inline ui/lib/app.js:99]: this isn't a number its a string

**Comment by progrium** [comment]
... did you test this? lol

**Comment by progrium** [comment]
see latest commit for working version

**Comment by maxknivets** [comment]
Vizgo keeps crashing on Linux, which made me think this was the same kind of crashes. Nonetheless, I should be careful next time I push changes. Sorry about that.

**Pull request merged**